### ISSUE TITLE
1.2.0 doesnt compile

### ISSUE THREAD

**Issue by EdJoPaTo** [bxparks/AceTimeClock#15]
When using PlatformIO (ESP32) with `bxparks/AceTimeClock@^1.2.0` the build fails:

```plaintext
In file included from .pio/libdeps/esp/AceTimeClock/src/AceTimeClock.h:29,
                 from src/main.cpp:7:
.pio/libdeps/esp/AceTimeClock/src/ace_time/clock/EspSntpClock.h: In member function 'virtual ace_time::acetime_t ace_time::clock::EspSntpClock::getNow() const':
.pio/libdeps/esp/AceTimeClock/src/ace_time/clock/EspSntpClock.h:52:22: error: 'secondsToCurrentEpochFromUnixEpoch64' is not a member of 'ace_time::LocalDate'
         - LocalDate::secondsToCurrentEpochFromUnixEpoch64();
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This is the listed Dependency Graph for `AceTimeClock`:

```plaintext
|-- AceTimeClock @ 1.2.0
|   |-- WiFi @ 2.0.0
|   |-- AceTime @ 2.0.0
|   |   |-- AceCommon @ 1.5.2
|   |   |-- AceSorting @ 1.0.0
|   |-- AceCommon @ 1.5.2
|   |-- AceSorting @ 1.0.0
```

Rolling back to `bxparks/AceTimeClock@~1.1.0` also does not work. Looks like the dependencies have no specified version range [here](https://github.com/bxparks/AceTimeClock/blob/ae638ab63b75223ba9480418e2f3a5ecee2f8432/library.properties#L10) so I wouldn't expect `~1.1` to work. [Looks like version ranges can be specified](https://arduino.github.io/arduino-cli/0.28/library-specification/#version-constraints) but I'm not sure they work in Arduino v1.

**Comment by bxparks** [comment]
Sorry about that. I fixed this on the develop branch yesterday, but I have not yet pushed a new release. Will do that very soon, like today. The `LocalDate::secondsToCurrentEpochFromUnixEpoch64()` should be `Epoch::secondsToCurrentEpochFromUnixEpoch64()` instead.

**Comment by EdJoPaTo** [comment]
Can confirm, ae638ab works as expected. I'll use it until a new release is there. Thank you!

**Comment by bxparks** [comment]
Fixed with new v1.2.1 release.
I also released EpoxyDuino v1.4.0 and EspMock v0.1, to allow the ESP8266 examples in AceTimeClock to run in GitHub Actions to check for compiler errors. Previously, they were not included because supporting the necessary ESP8266 mocks seemed like a lot of work, which it kinda was but not terribly. But now that the work is done, it feels good to have automated checking for the ESP8266 code paths.

**Comment by EdJoPaTo** [comment]
Looks like 1.2.1 is not yet known to PlatformIO: https://registry.platformio.org/libraries/bxparks/AceTimeClock/versions

Also the README dependencies state AceTime v1.8 (Line 151), isnt that 2.0 now?

**Comment by EdJoPaTo** [comment]
PlatformIO now also has the version 1.2.1.

**Comment by bxparks** [comment]
Thanks, updated the README.md. Released v1.2.2 with a bug fix in `NtpClock`.
